test/e2e/node/pod_resize.go

```diff
@@ -276,10 +272,6 @@ func doPodResizeTest() {
 	var podClient *framework.PodClient
 	var ns string
 
-	if !utilfeature.DefaultFeatureGate.Enabled(InPlacePodVerticalScalingFeature) {
```

Comment:
If we did this, doesn't the test fail under normal alpha mode? (i.e. featuregate is not enabled)

Reply:
It's supposed to be done this way: https://github.com/kubernetes/test-infra/pull/22826